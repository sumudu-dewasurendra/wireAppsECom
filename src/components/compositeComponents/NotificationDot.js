import React from 'react';
import { Text, View } from 'react-native';

const NotificationDot = ({
  count,
}) => {

  return (
    <>
        <View className={"w-6 h-6 bg-white rounded-full bg-red-500 justify-center items-center font-bold"}>
            <Text className={"text-white text-1xl text-center"}>{count > 9 ? '9+' : count}</Text>
        </View>
    </>
  );
}

export default NotificationDot;