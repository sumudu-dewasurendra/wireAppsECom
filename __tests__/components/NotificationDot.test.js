import React from 'react';
import renderer from 'react-test-renderer';
import NotificationDot from '../../src/components/NotificationDot';

test('renders correctly', () => {
  const tree = renderer.create(<NotificationDot count={0}  />).toJSON();
  expect(tree).toMatchSnapshot();
});