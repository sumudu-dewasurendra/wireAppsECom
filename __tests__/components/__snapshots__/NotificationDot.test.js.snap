// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "width": 24,
      },
      {
        "height": 24,
      },
      {
        "backgroundColor": "#fff",
      },
      {
        "borderBottomLeftRadius": 9999,
        "borderBottomRightRadius": 9999,
        "borderTopLeftRadius": 9999,
        "borderTopRightRadius": 9999,
      },
      {
        "backgroundColor": "#ef4444",
      },
      {
        "justifyContent": "center",
      },
      {
        "alignItems": "center",
      },
      {
        "fontWeight": "700",
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#fff",
        },
        {
          "textAlign": "center",
        },
      ]
    }
  >
    0
  </Text>
</View>
`;
